Source/UI/Screens/Identifier/IdentifierViewController.swift

```diff
@@ -170,6 +170,8 @@ class IdentifierViewController: IdentityUIViewController {
             showPhoneNumber()
             countryCode.text = prefilledPhoneComponents.countryCode
             phoneNumber.text = prefilledPhoneComponents.number
+        case .sharedWebCredentials:
+            fatalError()
```

Comment:
What does this stand for?

Reply:
This flow is impossible to hit (the view controller will never be presented with this login method)